common/src/crypto.ts

```diff
@@ -24,7 +24,20 @@ const encryptAlgo = {
 
 const format = "jwk";
 
-const stringToUIntArray = (str: string) => new TextEncoder().encode(str);
+export const stringEncode = (str: string): ArrayBuffer =>
```

Comment:
why the export?

Reply:
Because I forgot to delete that after testing 🤦